src/components/Garden/Stake/StakeManagement.tsx

```diff
@@ -54,7 +54,7 @@ const StakeManagement = React.memo(function StakeManagement() {
       {loading ? (
         <GardenLoader />
       ) : (
-        <LayoutGutter>
+        <LayoutGutter collapseWhenSmall={undefined}>
```

Comment:
If we add default value in `collapseWhenSmall` that its not necessary 
```suggestion
        <LayoutGutter>
```

Like below 
![image](https://user-images.githubusercontent.com/691510/161655464-758526c6-9a89-4982-ad7a-85cd28f061d4.png)

Reply:
<img width="1000" alt="Captura de ecrã 2022-04-05, às 11 14 47" src="https://user-images.githubusercontent.com/13184907/161732317-93184c1b-71d1-4f20-9413-1f74a9ce87e8.png">

Reply:
maybe that change is not in the version 1.0.17, WDYT?

Reply:
Don't you think it makes sense to be a boolean?

Reply:
Oh yeah you are right, i saw double question mark there ?? but it's not haha.

Reply:
So default should be false, right?

Reply:
Yep. I added already.